cmd/infra/powervs/destroy.go

```diff
@@ -492,8 +492,20 @@ func destroyPowerVsCloudConnection(ctx context.Context, options *DestroyInfraOpt
 			}
 		}
 	} else {
+
+		deleteCC := func(id string) error {
+			for i := 0; i < 5; i++ {
```

Comment:
any reason why not using builtin poll function as in other places?

Reply:
Builtin polling func is based on a timeout with unlimited retries. Want to limit the retry with a number. So used this logic.